primitives/account/src/account/staking_contract/validator.rs

```diff
@@ -356,19 +367,21 @@ impl StakingContract {
         store: &mut StakingContractStoreWrite,
```

Comment:
Just saw that the comment describing this method is an old copy-pasta from the deactivate method.